VideoWeb/VideoWeb/ClientApp/src/app/app-routing.module.ts

```diff
@@ -29,6 +29,6 @@ export const routes: Routes = [
 
 @NgModule({
     exports: [RouterModule],
-    imports: [RouterModule.forRoot(routes, { enableTracing: !environment.production })]
+    imports: [RouterModule.forRoot(routes, { enableTracing: environment.production })]
```

Comment:
This should only be for debugging. 

Is the angular environment always set to production?

Reply:
Thanks! i forgot to undo this.